pkg/registry/pod/etcd/etcd.go

```diff
@@ -146,15 +147,20 @@ func (r *BindingREST) New() runtime.Object {
 // Create ensures a pod is bound to a specific host.
 func (r *BindingREST) Create(ctx api.Context, obj runtime.Object) (out runtime.Object, err error) {
 	binding := obj.(*api.Binding)
-	err = r.assignPod(ctx, binding.PodID, binding.Host)
-	err = etcderr.InterpretCreateError(err, "binding", "")
+	err = r.assignPod(ctx, binding.PodID, binding.Host, binding.Annotations)
+	if err = etcderr.InterpretCreateError(err, "binding", ""); err != nil {
+		if _, ok := err.(*errors.StatusError); !ok {
```

Comment:
This is fine for now.  Bindings internal storage will be cleanup up over successive iterations so we don't need the perfect solution now.